x-pack/metricbeat/module/aws/sqs/_meta/fields.yml

```diff
@@ -12,7 +12,7 @@
       type: long
       description: >
         TThe number of messages in the queue that are delayed and not available for reading immediately.
-    - name: messages.not_visible
+    - name: message.not_visible
```

Comment:
I assume this means we don't check today if the fields are documented? I hope we get this coverage when we have the new "data tests".

Reply:
Yes! Will track this in https://github.com/elastic/beats/issues/11226